apps/fishing-map/features/workspaces-list/WorkspaceWizard.tsx

```diff
@@ -85,9 +88,15 @@ function WorkspaceWizard() {
     } else {
       updateMatchingAreas(inputValue)
     }
+    setInputSearch(inputValue)
```

Comment:
why do we need to replicate this? we already have the input value in [this param](https://github.com/GlobalFishingWatch/frontend/blob/add-analytics-mm/apps/fishing-map/features/workspaces-list/WorkspaceWizard.tsx#L143): `inputValue`
You could move the function below the hook to use it

Reply:
oh this PR wasn't done, but in this case.. the reason is because the inputValue has the full name of the selected object at the moment that an item is selected form the list, and I want to capture what the user type and what the user select @j8seangel

Reply:
got it, thanks!